public/modules/rewrite/my_bodies/edit_circles.html

```diff
@@ -0,0 +1,21 @@
+<table class="edit-circles-table">
```

Comment:
Yes, I thought about using the managed tables of wrapbootstrap, but I decided to postpone it because of lazyness then